src/app/modules/trade-companion/component/trade-notification/trade-notification.component.html

```diff
@@ -16,6 +16,7 @@
         <mat-icon class="incomingArrow" matTooltip="{{ 'trade-companion.trade-notification.incoming-trade' | translate }}">forward</mat-icon>
         <ng-container *ngTemplateOutlet="offerAndTimeTemplate"></ng-container>
         <mat-icon class="inviteToParty clickable" (click)="inviteToPartyClick()" matTooltip="{{ 'trade-companion.trade-notification.invite-to-party' | translate }}">add_circle_outline</mat-icon>
+        <mat-icon class="highlight clickable" (click)="highlightClick()" matTooltip="{{ 'trade-companion.trade-notification.highlight' | translate }}">search</mat-icon>
```

Comment:
I would like to suggest adding a setting to toggle this button on/off.  
Perhaps it's even a good idea to have settings for a couple of the other more niche buttons (such as whisper/whois/etc...) <<-- if this one is too much effort, it can be pushed back and moved to a separate issue.